src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -395,10 +398,25 @@ class LicenceService(
       ?: throw ValidationException("Staff with username $username not found")
 
     val updatedLicence = when (licenceEntity) {
-      is CrdLicence -> licenceEntity.submit(submitter as CommunityOffenderManager)
+      is CrdLicence -> {
+        if (!licenceHolderIsEligible(licenceEntity.nomsId)) {
```

Comment:
mmh.. could just move this check up front and do it once?

Reply:
I did initially, but as we're not blocking it for `VariationLicence`s, I thought it best to avoid unnecessary API calls to prisoner search